tasks/determine-image-expiration-parameter-task.yaml

```
@@ -0,0 +1,41 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: determine-image-expiration-parameter
+spec:
+  description: Determines the image expiration parameter, or sets to empty for release images.
```

Perhaps more accurate description would be
```suggestion
  description: Determines the image expiration parameter, or sets to empty for git-tagged builds.
```